Server/Interfaces/IChatServer.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Server.Interfaces
+{
+    public interface IChatServer
+    {
+
+        List<IClient> clients { get; set; }
```

Comment:
```suggestion
        IEnumerable<IClient> clients { get; set; }
```
have you tried to use it like this? can you tell what is better in this case?